### DISCUSSION TITLE
[Script request] Barcode Buddy

### DISCUSSION THREAD

**Discussion by T0ML** [community-scripts/ProxmoxVE#523] [Request script]
### Application Name

Barcode Buddy

### Website

https://github.com/Forceu/barcodebuddy

### Description

Barcode buddy ties in directly with the already created grocy script to allow for quick adding of new groceries via phone app or dedicated barcode scanning device.  Addition of this would complement the already made grocy script.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by bvdberg01** [comment]
Merged in PR: #2167
